### PR TITLE
move pkg/util/ipset inside pkg/proxy/ipvs

### PR DESCRIPTION
**Pull request by aroradaman**
#### What type of PR is this?
/kind cleanup

Follows code cleanup from https://github.com/kubernetes/kubernetes/pull/118296, https://github.com/kubernetes/kubernetes/pull/118296#issuecomment-1568714366

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/proxy/ipvs/ipset/ipset.go
Comment (uablrek) [inline pkg/proxy/ipvs/ipset/ipset.go:494]: Don't use "Sprintf", use a structured variable, and drop the underscore:
```go
	klog.ErrorS(validationError, "invalid protocol", "protocol", protocol, "supported protocols", []string{ProtocolTCP, ProtocolUDP, ProtocolSCTP})

```

**Review by uablrek — APPROVED** [review]

File: pkg/proxy/ipvs/ipset/ipset.go
Comment (danwinship) [reply pkg/proxy/ipvs/ipset/ipset.go]: I don't know the conventions for structured logging keys...

Comment (aroradaman) [reply pkg/proxy/ipvs/ipset/ipset.go]: I have updated structured logging keys with camel casing, **pkg/proxy/ipvs** uses the same convention and is quite common among other modules.

**Review by uablrek — APPROVED** [review]

**Review by uablrek — APPROVED** [review]

**Comment by uablrek** [comment]
@thockin pkg/util/ipset needs lgtm from sig-network-approvers (moved to proxy/ipvs by this PR)

**Pull request merged**
